main.js

```diff
@@ -455,7 +455,7 @@
     function fetchAllData() {
         Promise.all([
             fetchWPRDCData("Police", { limit: 250 }),
-            fetchWPRDCData("311", { limit: 250 }),
+            fetchWPRDCData("311"),
```

Comment:
Could we rm the arbitrary limitation on the others, too? Or do we need to have the work-around there?